### DISCUSSION TITLE
Windows installation - CLI only (from GitHub rep.?)

### DISCUSSION THREAD

**Discussion by adamsky-01** [Chia-Network/chia-blockchain#4736] [General]
Can chia-blockchain be installed without GUI under Windows 10? By cloning the GitHub Repository and opening in venv? Any guide for this option?

**Comment by lvcivs** [comment]
Have you checked this? https://github.com/Chia-Network/chia-blockchain/wiki/Quick-Start-Guide#windows

**Reply by adamsky-01** [reply]
Sure, I'm using CLI only and I can fully control all chia tasks this way. I'm just looking for a way to get the chia-blockchain installed on Windows 10 without GUI Windows installer.
